tests/src/Unit/Plugin/BundleSuggestion/BundleSuggestionTestBase.php

```diff
@@ -57,25 +57,12 @@ protected function setUp(): void {
     $entity_type_manager->method('getStorage')->willReturn($entity_storage);
 
     $file_system = $this->createMock(FileSystemInterface::class);
-    $file_system->method('uriScheme')->will($this->returnCallback([
-      $this,
-      'fileSystemUriSchemeCallback',
-    ]));
```

Comment:
So this is all baked in now in the new version?

Reply:
The method `uriScheme` is deprecated in 8.8 and its removed in 9. I think because of that deprecation, this mockup is no longer needed.